brion/types.h

```diff
@@ -53,16 +53,19 @@ class Mesh;
 class Morphology;
 class SpikeReport;
 class SpikeReportPlugin;
-
 class Synapse;
 class SynapseSummary;
 class Target;
 
-typedef vmml::vector< 2, int32_t > Vector2i; //!< A two-component int32 vector
-typedef vmml::vector< 3, float > Vector3f;   //!< A three-component float vector
-typedef vmml::vector< 4, float > Vector4f;   //!< A four-component float vector
-typedef vmml::vector< 3, double > Vector3d; //!< A three-component double vector
-typedef vmml::vector< 4, double > Vector4d; //!< A four-component double vector
+typedef boost::shared_ptr< Morphology > MorphologyPtr;
+typedef std::vector< MorphologyPtr > Morphologies;
```

Comment:
I kept the morphologies::load functions in brion. I think they still belong there.